lib/install.js

```diff
@@ -112,6 +116,10 @@ function install(argv, callback) {
       case 'I': // --skip-default-install
         jobConfig.env.STRONGLOOP_PM_SKIP_DEFAULT_INSTALL = 'true';
         break;
+      case 'M':
+        jobConfig.useMemoryDb = true;
+        jobConfig.dbUpdate = false;
```

Comment:
will a downgrade occur? if not, perhaps it should be fatal to use memory, if there is an existing sqlite3 db? maybe it is, I'm reading top-to-bottom, commenting as I go